src/main/ipc.ts

```diff
@@ -804,6 +839,39 @@ export default function setupIPCs(
     },
   );
 
+  ipcMain.removeHandler('getCurrentParryggTournaments');
+  ipcMain.handle('getCurrentParryggTournaments', getCurrentParryggTournaments);
+
+  ipcMain.removeHandler('getSelectedParryggTournament');
+  ipcMain.handle('getSelectedParryggTournament', getSelectedParryggTournament);
+
+  ipcMain.removeHandler('setSelectedParryggTournament');
+  ipcMain.handle(
+    'setSelectedParryggTournament',
+    (event: IpcMainInvokeEvent, slug: string) => {
+      setSelectedParryggTournament(slug);
+    },
+  );
+
+  ipcMain.removeHandler('setSelectedParryggSetChain');
+  ipcMain.handle(
+    'setSelectedParryggSetChain',
+    (event: IpcMainInvokeEvent, setChain: ParryggSetChain) => {
+      setSelectedParryggSetChain(setChain);
+    },
+  );
+
+  ipcMain.removeHandler('getSelectedParryggSet');
+  ipcMain.handle('getSelectedParryggSet', getSelectedParryggSet);
+
+  ipcMain.removeHandler('setSelectedParryggSetId');
+  ipcMain.handle(
+    'setSelectedParryggSetId',
+    (event: IpcMainInvokeEvent, setId: string) => {
+      setSelectedParryggSetId(setId);
+    },
+  );
```

Comment:
don't interleave with challonge methods, keep in contiguous blocks as much as possible